pages/partials/question.ejs

```diff
@@ -70,13 +70,20 @@
         <% } else { %>
             <% throw Error('unknown question_context: ' + question_context); %>
         <% } %>
-      <% if (authz_data.mode == 'Exam') { %>
-        <div class="float-sm-right">
-          <button class="btn btn-xs btn-secondary" type="button" data-toggle="modal" data-target="#iqqrcode" aria-expanded="false" aria-controls="iqqrcode">
-            Proctor <i class="fa fa-question-circle"></i>
+        <% if (authz_data.mode == 'Exam') { %>
+        <div class="float-sm-right ml-1">
+          <button class="btn btn-xs btn-light" type="button" data-toggle="modal" data-target="#iqqrcode" aria-expanded="false" aria-controls="iqqrcode">
+            Proctor <i class="fa fa-question-circle" aria-hidden="true"></i>
           </button>
         </div>
-      <% } %>
+        <% } %>
+        <% if (locals.workspaceUrl) { %>
+        <div class="float-sm-right ml-1">
+          <a class="btn btn-xs btn-light" target="_blank" href="<%= locals.workspaceUrl %>">
+            Open workspace <i class="fa fa-external-link-alt" aria-hidden="true"></i>
```

Comment:
Nit: e should consider using the same icon in the `pl-workspace` element, and possibly using a different icon - https://fontawesome.com/icons/laptop-code?style=solid or https://fontawesome.com/icons/code?style=solid might be good options.

Reply:
Good point, switched both buttons to `fa-laptop-code` in 8dad16c705a93dc1806f82063c18efda39e17c09